pages/instructorAssessmentDownloads/instructorAssessmentDownloads.sql

```diff
@@ -72,8 +72,7 @@ FROM
     JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
     LEFT JOIN group_configs AS gc ON (gc.assessment_id = a.id)
     LEFT JOIN groups AS g ON (g.id = ai.group_id AND g.group_config_id = gc.id)
-    LEFT JOIN group_users AS gu ON (gu.group_id = g.id)
-    JOIN users AS u ON (u.user_id = ai.user_id OR u.user_id = gu.user_id)
+    LEFT JOIN users AS u ON (u.user_id = ai.user_id)
```

Comment:
Checking my understanding: I believe this means that `uid`, `uin`, `name`, and `role` will be null for group assessment instances, but this is fine because of this line?

https://github.com/PrairieLearn/PrairieLearn/blob/f207b375c361055840a12d65c5efdca1a6cbb9c9/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.js#L126-L128

Reply:
Yup, that's basically it. For all files that use these 3 queries, the UID/UIN/Name/Role are not used at all as columns or in any processing. The files where rows are per user (instead of per group) use a different query that remains joined with group_users:
https://github.com/PrairieLearn/PrairieLearn/blob/d149ab41ec6fbedc0b50c2c1dd91d38a22a29644/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.sql#L29-L30